flowforge/input/BoundaryConditions.py

```diff
@@ -132,6 +132,64 @@ def _convertUnits(self, uc: UnitConverter) -> None:
             else:
                 raise Exception("Unknown enthalpy BC type: " + self.type_outlet)
 
+class VoidBC:
+    """Class for void fraction BC
+
+    Attributes
+    ----------
+    mdot : float
+        Mass flow rate [kg/s]
+    void_fraction : float
+        Void fraction [-]
+    """
+
+    def __init__(self, inlet: dict = None, outlet: dict = None):
+        assert inlet or outlet
+
+        self._type_inlet = None
+        self._val_inlet = None
+        if inlet:
+            if "mdot" in inlet:
+                self._type_inlet = "mdot"
+                self._val_inlet = inlet["mdot"]
+            else:
+                self._type_inlet = "void_fraction"
+                self._val_inlet = inlet["void_fraction"]
+
```

Comment:
ok, so as of now, you're not handling viable errors.  You need to add an assertions to make sure a user provides one of the valid types (i.e. `mdot` or `void_fraction`), doesn't provide both, doesn't specify both inlet and outlet, and that `mdot > 0.` or `0. <= void_fraction <= 1.`.

Reply:
![image](https://github.com/user-attachments/assets/bba95c64-82af-4472-ac94-7a4a18049cd1)

for MMBC the mdot can be anything but 0

Reply:
![image](https://github.com/user-attachments/assets/7732a65e-2277-47dd-91a5-1925823ed264)

Current